problem_builder/tests/unit/test_problem_builder.py

```diff
@@ -251,7 +251,7 @@ def test_get_standard_results_calls_get_option(self):
 class TestMentoringBlockJumpToIds(unittest.TestCase):
     def setUp(self):
         self.service_mock = Mock()
-        self.runtime_mock = Mock()
+        self.runtime_mock = Mock(replace_urls=lambda text: text)
```

Comment:
The actual replacing of URLs is done by the runtime and we can't really test it; we just need to make sure `self.runtime.replace_urls(...)` gets called at some point by the XBlock if available.

I am guessing this change is just making the mock API better match the runtime API, to avoid an error message that runtime.replace_urls(text) does not exist on the mock runtime. The code in our `expand_static_url` method does check `hasattr(self.runtime, 'replace_urls')` before trying to call that method, but because it's a Mock object, `hasattr` always returns `True`.